index.html

```diff
@@ -354,7 +354,7 @@ <h2 class="contact__title fw-700 fz-40 lh-52">Contact me</h2>
           feature that you need built or a project that needs coding. I'd love
```

Comment:
- [ ] Kindly change the contrast to be at least `4.5` for these failed elements because they failed in the contract check
![image](https://user-images.githubusercontent.com/24703171/166472610-5905820d-7519-4a10-a101-7d8acc68ed44.png)
You can check the contrast for your website using this [link](https://www.aditus.io/button-contrast-checker/)

Reply:
If we implement the changes requested above that end up passing the original figma design!
Please advice how best we proced?